### PR TITLE
Add a note for SELinux to change the SELinux context of the server and agent binaries

### PR DESCRIPTION
**Pull request by eve-bright**
If you open a PR that needs to go into a current version, you need to *cherry-pick your commit from dev over to the current version branch*. Only then will the proper builds that generate html/pdf be run. But beware: Docs will be generated but not published automatically!

- [x] N/A - or - I have added the appropriate "cherry-pick-to" labels to this PR so I don't forget to do this later!

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Review by alexnb — COMMENTED** [review]
During the session on Friday we found out that also the agent binary needed to be marked as `bin_t`.

**Comment by eve-bright** [comment]
> During the session on Friday we found out that also the agent binary needed to be marked as `bin_t`.

Ah! Of course - I can see that in your card now too.

File: modules/ROOT/pages/addition/agent-installation/manual.adoc [hunk 1]
```diff
@@ -41,10 +41,23 @@ The following steps are required to install an agent:
...
+** This step is only required if using *SELinux*.
+Run the following commands to permanently change the SELinux context of the agent binary:
++
+[source, terminal, role=noheader]
+----
+sudo semanage fcontext -a -t bin_t <EXTRACTED_DIRECTORY_PATH>/bin/agent
+sudo restorecon -v <EXTRACTED_DIRECTORY_PATH>/bin/agent
+----
++
+<EXTRACTED_DIRECTORY_PATH> should be replaced with the path to the extracted diretory
```
Comment (AlexicaWright) [inline modules/ROOT/pages/addition/agent-installation/manual.adoc]:
```suggestion
<EXTRACTED_DIRECTORY_PATH> should be replaced with the path to the extracted directory.
```

File: modules/ROOT/pages/addition/agent-installation/manual.adoc [hunk 2]
```diff
@@ -41,10 +41,23 @@ The following steps are required to install an agent:
 
 NOTE: If downloading agents is not a option in some environments, NOM agents are also bundled into Neo4j packages which can be used for the installation as described xref:./install-packaged.adoc[here]
 
-** Extract the archive:
+** Extract the archive
+
+** This step is only required if using *SELinux*.
+Run the following commands to permanently change the SELinux context of the agent binary:
```
Comment (AlexicaWright) [inline modules/ROOT/pages/addition/agent-installation/manual.adoc]:
```suggestion
To permanently change the SELinux context of the agent binary, run the following commands:
```

File: modules/ROOT/pages/addition/agent-installation/self-registered.adoc [hunk 3]
```diff
@@ -20,6 +20,17 @@ NOTE: If downloading agents is not an option in some environments, NOM agents ar
 
 ** Extract the archive:
 
+** This step is only required if using *SELinux*.
+Run the following commands to permanently change the SELinux context of the agent binary:
```
Comment (AlexicaWright) [inline modules/ROOT/pages/addition/agent-installation/self-registered.adoc]:
```suggestion
To permanently change the SELinux context of the agent binary, run the following commands:
```

File: modules/ROOT/pages/addition/agent-installation/self-registered.adoc [hunk 4]
```diff
@@ -20,6 +20,17 @@ NOTE: If downloading agents is not an option in some environments, NOM agents ar
...
++
+[source, terminal, role=noheader]
+----
+sudo semanage fcontext -a -t bin_t <EXTRACTED_DIRECTORY_PATH>/bin/agent
+sudo restorecon -v <EXTRACTED_DIRECTORY_PATH>/bin/agent
+----
++
+<EXTRACTED_DIRECTORY_PATH> should be replaced with the path to the extracted diretory
```
Comment (AlexicaWright) [inline modules/ROOT/pages/addition/agent-installation/self-registered.adoc]:
```suggestion
<EXTRACTED_DIRECTORY_PATH> should be replaced with the path to the extracted directory.
```

File: modules/ROOT/pages/installation/server.adoc [hunk 5]
```diff
@@ -89,6 +89,20 @@ To stop the NOM Server, use the command:
 sudo systemctl stop neo4j-ops-manager-server
 ----
 
+[NOTE]
+====
+*For SELinux*
+
+The SELinux context of the server binary must be changed so that it can be executed from the service. 
+Run the following commands to permanently change the SELinux context of the file:
```
Comment (AlexicaWright) [inline modules/ROOT/pages/installation/server.adoc]:
```suggestion
To permanently change the SELinux context of the file, run the following commands:
```

**Review by AlexicaWright — COMMENTED** [review]
Some editorial comments, otherwise it LGTM.

**Review by alexnb — APPROVED** [review]
I tested the commands on GCP with CentOS 7. They worked (I had to `yum install policycoreutils-python` to be able to call `semanage`).

Ironically I couldn't reproduce missing `bin_t` although `sestatus` was on, this was strange. But executing agent and server always worked.

**Pull request merged**